aws-gitlab-main.yml

```diff
@@ -24,7 +24,9 @@ Resources:
                   - 's3:GetObject'
                   - 's3:DeleteObject'
                   - 's3:PutObjectAcl' 
-                  Resource: 'arn:aws:s3:::aws-gitlab-bucket'
+                  Resource: 
+                    gl-s3-bucket:
```

Comment:
bruh tf? this was new one i have not seen before.

Great the bucket as another resource like the role, the profile and !Ref on the bucket for Resource piece of the role